src/workflows/trades.ts

```diff
@@ -1,16 +1,16 @@
-import { Signer } from '@ethersproject/abstract-signer';
-import { StarkWallet } from '../types';
+import { L1Signer, L2Signer } from '../types';
 import { GetSignableTradeRequest, TradesApi } from '../api';
-import { serializeSignature, sign, signRaw } from '../utils';
+import { signRaw } from '../utils';
 
 export async function createTradeWorkflow(
-  signer: Signer,
-  starkWallet: StarkWallet,
+  l1Signer: L1Signer,
+  l2Signer: L2Signer,
+  l2Message: string,
```

Comment:
there should be any l2Message's from the params

Reply:
Removed l2Message, right now using payloadHash